lib/server.js

```diff
@@ -132,7 +132,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 async function _handleNotification({req}) {
   const {peer} = req.body;
   const localPeerId = config.server.baseUri +
-    '/consensus/continuity2017/voters/' + req.params.voterId;
+    '/consensus/continuity2017/peers/' + req.params.peerId;
```

Comment:
If we add `encodeURIComponent` here and elsewhere, it will prepare us for using DIDs as peer IDs:

```suggestion
    `/consensus/continuity2017/peers/${encodeURIComponent(req.params.peerId)}`;
```